packages/gafl-webapp-service/src/pages/errors/client-error.njk

```diff
@@ -18,18 +18,14 @@
 {% block content %}
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">
-          A problem has occurred
-        </h1>
-
             {% call govukFieldset({
               legend: {
-                text: title,
-                classes: "govuk-fieldset__legend--m govuk-!-margin-bottom-3",
+                text: "A problem has occurred",
+                classes: "govuk-fieldset__legend--l govuk-!-margin-bottom-6",
                 isPageHeading: true
               }
             }) %}
-
+            <h2 class="govuk-heading-m">{{ title }}</h2>
```

Comment:
This looks like we'll end up with 'A problem has occurred' repeated two or three times on the page - or am I wrong?

Reply:
If the statusCode is not 400, 401, 403, 404 then yes, title would be 'A problem has occurred' and be repeated twice. However, this is existing behaviour.

Reply:
The pageTitle block is the text shown in browser's title bar